### ISSUE TITLE
Length of reaction arrow mismatch and empty line

### ISSUE THREAD

**Issue by tomyan112** [Chem4Word/Version3-2#34]
**Describe the bug**
Length of reaction arrow after rendering does not match with the editor
There is an empty line under 1.O₃. I only typed one line.

**To Reproduce**
![image](https://user-images.githubusercontent.com/9064351/156907671-5e0466e1-797d-4eb4-8135-7fe8484905b5.png)

Edit: Put back markdown removed by reportee
**CML Data**
```xml
<cml:cml xmlns:conventions="http://www.xml-cml.org/convention/" xmlns:cml="http://www.xml-cml.org/schema" xmlns:cmlDict="http://www.xml-cml.org/dictionary/cml/" xmlns:nameDict="http://www.xml-cml.org/dictionary/cml/name/" xmlns:c4w="http://www.chem4word.com/cml" conventions="convention:molecular">
  <c4w:customXmlPartGuid>493ceaca9d9947ae9a567fa1ad920fb4</c4w:customXmlPartGuid>
  <cml:reactionScheme id="rs1">
    <cml:reaction id="r1" c4w:head="1291.0135,1200.0" c4w:tail="1108.9865,1200.0">
      <c4w:reagents>
        <FlowDocument mc:Ignorable="c4w" xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" xmlns:c4w="http://www.chem4word.com/cml" xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
          <Paragraph>
            <Run>1. O</Run>
            <Run FontSize="18" BaselineAlignment="Subscript">3</Run>
            <LineBreak />
          </Paragraph>
        </FlowDocument>
      </c4w:reagents>
      <c4w:conditions>
        <FlowDocument mc:Ignorable="c4w" xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" xmlns:c4w="http://www.chem4word.com/cml" xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
          <Paragraph>
            <Run>2. Zn, H</Run>
            <Run FontSize="18" BaselineAlignment="Subscript">2</Run>
            <Run>O</Run>
            <LineBreak />
          </Paragraph>
        </FlowDocument>
      </c4w:conditions>
    </cml:reaction>
  </cml:reactionScheme>
</cml:cml>
```

**Screenshots**

**Desktop (please complete the following information):**
Chem4Word 2022 Beta 2

**Comment by MikeWilliams-UK** [comment]
> There is an empty line under 1.O₃. I only typed one line.

@tomyan112 this bug has two parts the second has been fixed in our internal version which will become V3.2.3

**Comment by MikeWilliams-UK** [comment]
> Length of reaction arrow after rendering does not match with the editor

@deadlyvices can you take a look at this please?

**Comment by MikeWilliams-UK** [comment]
Included in V3.2.3 Beta 3
